deps.bzl

```diff
@@ -826,13 +826,13 @@ def install_buildbuddy_dependencies(workspace_name = "buildbuddy"):
         sum = "h1:DshxFxZWXUcO0xX476VJC07Xsr6ZCBVRHKZ93Oh7Evo=",
         version = "v1.6.0",
     )
-
     go_repository(
```

Comment:
nit: can this be removed? (I think gazelle might not delete stuff from `deps.bzl`, so may be worth deleting and rerunning buildfix)